pkg/gadgets/trace/dns/types/dns.go

```diff
@@ -19,12 +19,22 @@ import (
 	eventtypes "github.com/inspektor-gadget/inspektor-gadget/pkg/types"
 )
 
+type DNSPktType string
+
+const (
+	DNSPktTypeQuery    DNSPktType = "Q"
+	DNSPktTypeResponse DNSPktType = "R"
+)
+
 type Event struct {
 	eventtypes.Event
 
-	PktType string `json:"pktType,omitempty" column:"type,minWidth:7,maxWidth:9"`
-	QType   string `json:"qtype,omitempty" column:"qtype,minWidth:5,maxWidth:10"`
-	DNSName string `json:"name,omitempty" column:"name,width:30"`
+	ID         string     `json:"id,omitempty" column:"id,width:2,fixed,hide"`
+	Qr         DNSPktType `json:"qr,omitempty" column:"qr,width:2,fixed"`
```

Comment:
> I tested it, but something is not working with the columns format. The following command doesn't produce any output:
> 
> ```bash
> ./kubectl-gadget trace dns -A
> ```
> 
> But using `-o json` works properly.
> 

The issue seems related to the `DNSPktType` type in the columns library. Changing this field to use the `string` type makes things work again. cc @flyth